### ISSUE TITLE
PreferenceFragment Wrapper?

### ISSUE THREAD

**Issue by semireg** [bluelinelabs/Conductor#282]
I've read up on the wrapping of DialogFragment via #112 and was hoping I could learn more about wrapping legacy fragment-based code. Is there an example of how to do this?

**Comment by inorichi** [comment]
Just a week ago I wrote a lib to create preference screens with conductor [here](https://github.com/inorichi/conductor-support-preference). It's using preferences v7 but replacing fragments with controllers. I also used the dialog controller from the already opened dialogs PR here.

There are a few things I removed from the default `PreferenceFragmentCompat` (two listeners), but AFAIK everything is  working.

You'll need to add `<item name="preferenceTheme">@style/PreferenceThemeOverlay.Material</item>` to your theme (unless you want to create your own). Also don't include preferences v14, as `MultiSelectListPreference` (the only valuable thing there) is also available in the lib.
